pongo.sh

```diff
@@ -30,22 +31,24 @@ function globals {
   fi
 
   # regular Kong Enterprise images repo (tag is build as $PREFIX$VERSION$POSTFIX).
-  # Set credentials in $BINTRAY_APIKEY and $BINTRAY_USERNAME
-  KONG_EE_REPO="kong-docker-kong-enterprise-edition-docker.bintray.io"
-  KONG_EE_TAG_PREFIX="kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition:"
+  KONG_EE_TAG_PREFIX="kong/kong-gateway:"
   KONG_EE_TAG_POSTFIX="-alpine"
 
+  # all Kong Enterprise images repo (tag is build as $PREFIX$VERSION$POSTFIX).
+  KONG_EE_PRIVATE_TAG_FALLBACK_PREFIX="kong/kong-gateway-private:"
+  KONG_EE_PRIVATE_TAG_FALLBACK_POSTFIX="-alpine"
+
   # regular Kong CE images repo (tag is build as $PREFIX$VERSION$POSTFIX)
   KONG_OSS_TAG_PREFIX="kong:"
   KONG_OSS_TAG_POSTFIX="-alpine"
+
   # unoffical Kong CE images repo, the fallback
   KONG_OSS_TAG_FALLBACK_PREFIX="kong/kong:"
   KONG_OSS_TAG_FALLBACK_POSTFIX=
 
   # Nightly EE images repo, these require to additionally set the credentials
-  # in $NIGHTLY_EE_APIKEY and $NIGHTLY_EE_USER
-  NIGHTLY_EE_DOCKER_REPO="registry.kongcloud.io"
-  NIGHTLY_EE_TAG="registry.kongcloud.io/kong-ee-dev-master:latest"
+  # in $DOCKER_USERNAME and $DOCKER_PASSWORD
+  NIGHTLY_EE_TAG="kong/kong-gateway-internal:next-2.4.x.x-nightly-alpine"
```

Comment:
Based on the branching strategy we should make this point to the `master` nightly instead.

```suggestion
  NIGHTLY_EE_TAG="kong/kong-gateway-internal:master-nightly-alpine"
```